packages/kbn-ui-shared-deps-npm/BUILD.bazel

```diff
@@ -30,6 +30,7 @@ NPM_MODULE_EXTRA_FILES = [
 
 RUNTIME_DEPS = [
   "//packages/elastic-datemath",
+  "//packages/kbn-custom-events",
```

Comment:
@afharo could you elaborate? Not sure I'm following

Reply:
Sorry! I think that we need to move this line to the `*-src/BUILD.bazel` file instead of the current `*-npm/BUILD.bazel`. AFAIK, `*-npm` is used for `npm install`ed while `*-src` is used for packages in the monorepo. Or so I understood from my conversation with @mistic.

Although the previous line in `*-npm` declares `"//packages/elastic-datemath"` as a dependency in this module 🤷 . 

@mistic @spalger would you mind shedding some light here?